TranslatorReasonerAPI.yaml

```diff
@@ -632,15 +632,27 @@ components:
             Corresponds to the map key identifier of the
             object concept node anchoring the query filter
             pattern for the query relationship edge.
-        constraints:
+        attribute_constraints:
           type: array
           description: >-
-            A list of contraints applied to a query edge.
+            A list of attribute contraints applied to a query edge.
             If there are multiple items, they must all be true (equivalent
             to AND)
           items:
             $ref: '#/components/schemas/QueryConstraint'
```

Comment:
Now that we have the ability to constrain queries based on values of Attributes and Qualifiers, consider renaming the QueryConstraint object 'AttributeConstraint' or 'AttributeQueryConstraint'?  And then name the object that defines a constraint based on a Qualifier value 'QualifierConstraint' or 'QualifierQueryConstraint'.  This would make naming more clear and consistent.

Reply:
I think we can use `AttributeConstraint` and `QualifierConstraint` to distinguish the two kinds of constrains.